### ISSUE TITLE
[TW-1273] Words in task descriptions and annotations are cut-off after a hyphen

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1307]
_Elias Probst on 2014-02-21T19:31:18Z says:_ 

 Words containing a hyphen are cut off starting at the hyphen in task descriptions and annotations as shown in the following example where words like `re-evaluated` are shortened to `re`:

```
elias(at)moria ~ % task add Check, whether requirements for project-foo still need to be re-evaluated once project-bar is finished
Created task 1.
elias(at)moria ~ % task 1 annotate Also re-evalute the requirements for project-blah
Annotating task 1 'Check, whether requirements for project still need to be re once project is finished'.
Annotated 1 task.
elias(at)moria ~ % task long

ID Added      Age Description                                                                         
 1 2014-02-21 29s Check, whether requirements for project still need to be re once project is finished
                    2014-02-21 Also re the requirements for project

1 task
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-02-21T19:31:18Z
Modified: 2014-06-08T13:04:26Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-02-22T03:48:41Z says:_ 

 Thanks Elias, this should be fixed in 2.4.0.

As a workaround you can use the '--' parser override like this:

```
task add -- Check, whether requirements for project-foo still need to be re-evaluated once project-bar is finished
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-06-08T13:04:26Z says:_ 

 Solved in 2.4.0 without the need for the '--' terminator.
